stl/src/vector_algorithms.cpp

```diff
@@ -166,6 +272,7 @@ void* __cdecl __std_swap_ranges_trivially_swappable(
 
 } // extern "C"
 
+#if !defined(_M_ARM64)

```

Comment:
Just `#ifndef`. Even if you anticipate some additions in the future - change to `#if` can be done when the said additions are done.